moderation-commands/infractions.md

```diff
@@ -12,17 +12,17 @@ By default, a user is required to have the following permissions to use this com
 
 * Moderate Members
 
-For more information on editing permission requirements for specific users/roles, refer to the [permissions tutorial](<linkToPermissionsTutorial>)
+For more information on editing permission requirements for specific users/roles, refer to the [permissions tutorial](/start-up/permission-tutorial.md)
```

Comment:
there was no need to replace this, it will be done in one go with a find-and-replace-all